packages/mask/src/extension/popups/hook/useQueryIsBound.ts

```diff
@@ -0,0 +1,11 @@
+import { useAsyncRetry } from 'react-use'
+import { NextIDProof } from '@masknet/web3-providers'
+import { NextIDPlatform } from '../../../../../plugin-infra/node_modules/@masknet/shared-base/dist'
+
+export function useQueryIsBound(account?: string) {
+    const { value: infos } = useAsyncRetry(async () => {
```

Comment:
You did not use the retry function. Change to `useAsync`

Reply:
ok